### ISSUE TITLE
Week schedule vs manual mode

### ISSUE THREAD

**Issue by martijn-brant** [stefboerrigter/wavinsentio-ha#11]
Hi,

First, great job thusfar, seems to work great on my Sentio install. I noticed one thing you may want to avoid.

Most of my rooms have underfloor heating, are  in manual mode and set to a steady temperature. However, there are two rooms with radiators and they have a weekly schedule. I notice if I try and adjust one of the rooms on a schedule, Sentio ignores it. When I disable the schedule via the remote screen, switching it to manual mode, it does accept the new temperature set via HA.

Perhaps include a setting/dropdown on the state the room is in? Manual/Eco/Comfort/Extra Comfort/Weekly schedule?

I think it's:

"Room mode R/W Holding register 00117 
0 SCHEDULE
1 MANUAL
In SCHEDULE mode, the “Room temperature setpoint” is not used 
and the room temperature is controlled by scheduler."

Not quite sure how the modbus knows how to set eco/comfort/extra comfort.

**Comment by stefboerrigter** [comment]
I've noticed that rooms in schedule have some sort of timing issue with the latest firmware release.
When i try enough iterations at some point the new setpoint temperature is accepted,

The root cause for this problem is that there are either two options via modbus:
1. use weekschedule mode (and don't set a temperature) 
2. use manual mode and set a temperature manually. 

I've been informed that there will be a new FW release soon that covers this from the device perspective. That will mean that with the new release (expected Q3) the Sentio system will take care of what mode it is in, and will set the temperature.
That will mean it will behave more like the mobile app. 
That would result in when a room is in SCHEDULE mode, it will accept a new temperature as an temporary override untill the next schedule change is planned.

**Comment by martijn-brant** [comment]
Indeed, I'm not quite sure which version I'm on per component but the integration is telling me FW 14.0. But like I suggested, would it be possible to add a dropdown/toggle for scheduled/manual/eco/com/extrcom modes per room? This way I could toggle it manually. Perhaps the preset feature of climate entities is a good place for that? Not sure.

Btw. I believe your Sentio integration has the same name as another Sentio integration that's cloud based. My HA got very confused when using the both at same time. Perhaps consider adding modbus to your integration / package name?

**Comment by stefboerrigter** [comment]
Off course it is possible, not sure time wise when i'll be able to add it, will look into it for sure. 

Regarding the name, you are correct, it is copied from that integration and modified. Will also pick this up.

**Comment by stefboerrigter** [comment]
i have briefly checked the implementation, currently i see there is already a drop-down option but it just doesn't use the ECO / Comfort and extra comfort settings.
![image](https://github.com/stefboerrigter/wavinsentio-ha/assets/4964819/8e68e80d-92ad-491f-9773-d16abd601226)

The Automatisch (dutch) is Automatic and speaks for itself.
When setting the temperature or setting it from Automatic to OFF (Uit) it will be in manual mode, allowing you to toggle the temperature. 

Note; name change applied, see commit's above.

**Comment by martijn-brant** [comment]
Well I was thinking the heat/cool was actually used when doing heating or cooling. My system (heatpump based) has an input towards Sentio. If the outside temperature is too hot (measured by HP), it will signal Sentio that it will start pumping cold water rather than hot water.

So the cooling or heating status is mostly for when a room is too cold and it needs actual heat or too hot and need cooling with cold water, right? That's how it used to work with the other cloud based integration. All my climate entities would switch over from setting a minimum temp of 20c and going into having a max temperature of 22c.

**Comment by martijn-brant** [comment]
> Note; name change applied, see commit's above.

@stefboerrigter, should the json manifest also be updated? When I want to download the new version from the main branch, HACS tells me:

>When downloaded this will be located in '/config/custom_components/wavinsentio'

So it would again overwrite the cloud-based version.

`  "domain": "wavinsentio",
  "name": "Wavin Sentio",`

Pressing download in HACS on main branch even hangs the HACS download process.

**Comment by stefboerrigterWavin** [comment]
Hi Martijn,
Sorry to have ignored this for such a time, i cannot find time to fix it. if you want feel free to create a pull request. i'll do my best to find some time soon but it will remain limited unfortunatly.
